ROADMAP.md

```diff
@@ -27,9 +17,23 @@
 7. Switch overlay service from angular animations to native Web Animations [#8780](https://github.com/IgniteUI/igniteui-angular/issues/8780)
 8. Suggestion for igx-grid group by row template [#12752](https://github.com/IgniteUI/igniteui-angular/issues/12752)
 9. Way to filter auto-generated columns [#12577](https://github.com/IgniteUI/igniteui-angular/issues/12577)
+10. IgxDateRangePicker UX improvement [#11994](https://github.com/IgniteUI/igniteui-angular/issues/11994)
+11. Resizing [IgxDrag] [#6283](https://github.com/IgniteUI/igniteui-angular/issues/6283)
 
 # Previous Milestone
 
+## Milestone 27,  version 16.1 (Released Oct 5th, 2023) [Release Blog](https://www.infragistics.com/community/blogs/b/infragistics/posts/ignite-ui-for-angular-16-1-0)
+
+1. **[DONE]** igniteui-angular as standalone component exports [#12698](https://github.com/IgniteUI/igniteui-angular/issues/12698)
```

Comment:
Your list here is wrong. Seems like it's the `16.0.0` list. There's a number of tree-shaking improvements, the size mixin is added and the button group gets a new selection mode with `16.1.0`. Maybe the release doesn't have the correct changelog and some stuff got omitted? Please review and update here and the release if needed!

https://github.com/IgniteUI/igniteui-angular/releases/tag/16.1.0

Reply:
@radomirchev This comment is still relevant